src/java/edu/slu/tpen/entity/image/Canvas.java

```diff
@@ -254,63 +255,63 @@ public static JSONArray getLinesForProject(Integer projectID, String canvasID, I
+//                  pageList.add(JsonHelper.buildPage(f, "v3"));
+//            }
+//            System.out.println("Put all canvas together");
+//            annotationPage.put("resources", pageList);
+//            annotations = getAnnotationSet(projectID,folioNumber);
 //            System.out.println(Arrays.toString(annotations));
+
         
-        }
-        annotationPage.put("items",annotations);
-        //annotationList.element("@context", "http://iiif.io/api/presentation/2/context.json");
-        //annotationList.element("testing", "msid_creation");
-        
+//        }
+        annotationPage.put("items",annotations); 
         Transcription[] lines;
         lines = getProjectTranscriptions(projectID, folioNumber); //Can return an empty array now.
         int numberOfLines = lines.length;
         List<Object> resources = new ArrayList<>();
-        //System.out.println("How many lines?   "+numberOfLines);
         for (int i = 0; i < numberOfLines; i++) { //numberOfLines can be 0 now.
             if (lines[i] != null) {
-                //System.out.println("On line "+i);
                 dateString = "";
-                //when it breaks, it doesn't get this far
-                //System.out.println(lines[i].getLineID() + " " +lines[i].getDate().toString());
                 int lineID = lines[i].getLineID();
                 Map<String, Object> lineAnnot = new LinkedHashMap<>();
                 String lineURI = "line/" + lineID;
                 String annoLineID = getRbTok("SERVERURL") + "line/" + lineID;
-                //lineAnnot.put("@id", lineURI);
                 lineAnnot.put("id", annoLineID);
-                lineAnnot.put("_tpen_line_id", lineURI);
-                lineAnnot.put("type", "AnnotationPage");
-                lineAnnot.put("motivation", "oad:transcribing");
-                lineAnnot.put("resource", buildQuickMap("@type", "cnt:ContentAsText", "cnt:chars", encoder().decodeForHTML(lines[i].getText())));
-                lineAnnot.put("on", format("%s#xywh=%d,%d,%d,%d", canvasID, lines[i].getX(), lines[i].getY(), lines[i].getWidth(), lines[i].getHeight()));
+                lineAnnot.put("type", "Annotation");
+                lineAnnot.put("motivation", "transcribing");
+		
+		// Annotation body
+		Map<String, String> body = JsonHelper.buildAnnotationBody("cnt:ContentAsText", encoder().decodeForHTML(lines[i].getText()));
+		lineAnnot.put("body", body);
+		lineAnnot.put("target", canvasID);
+		// `target` replaces `on` from version 2 and it seems like they don't want the dimensions on the canvas url
+		// lineAnnot.put("target", format("%s#xywh=%d,%d,%d,%d", canvasID, lines[i].getX(), lines[i].getY(), lines[i].getWidth(), lines[i].getHeight()));
+
+
+		// All these properties below are from version 2 annotations, but it seems like they are project-specific and therefore should be here as well
+		lineAnnot.put("_tpen_line_id", lineURI);
```

Comment:
good. We will want to remove these later, but for the legacy version, we should retain them.